rosbag2_compression/test/rosbag2_compression/test_zstd_compressor.cpp

```diff
@@ -32,6 +36,23 @@ namespace
 {
 constexpr const char kGarbageStatement[] = "garbage";
 constexpr const int kDefaultGarbageFileSize = 10;  // MiB
+constexpr const size_t kExpectedCompressedDataSize = 976; // manually calculated, could change
```

Comment:
```suggestion
constexpr const size_t kExpectedCompressedDataSize = 976;  // manually calculated, could change
```
This is causing linters to fail.